plone-5.2.x.cfg

```diff
@@ -16,21 +16,42 @@ package-name =
 newest = false
 show-picked-versions = true
 
+plone-user = admin:admin
+
 [versions]
 # FIXME: This can be removed if Plone folks add the pin to Plone official cfgs.
 # https://github.com/collective/buildout.plonetest/issues/25
 zc.recipe.cmmi = 1.3.6
 
 [instance]
 recipe = plone.recipe.zope2instance
-user = admin:admin
-http-address = 8080
+user = ${buildout:plone-user}
+wsgi = on
 eggs =
     Plone
     plone.app.upgrade
     ${buildout:package-name}
     ${buildout:eggs}
+# zeo-client = on
+# zeo-address = 8100
+# shared-blob = on
+environment-vars =
+    zope_i18n_compile_mo_files true
 zcml =
     ${buildout:package-name}
-environment-vars = zope_i18n_compile_mo_files true
+
+[instance:python27]
+recipe = plone.recipe.zope2instance
+user = ${buildout:plone-user}
+http-address = 8080
+eggs =
+    Plone
+    plone.app.upgrade
+    ${buildout:package-name}
+    ${buildout:eggs}
 deprecation-warnings = on
```

Comment:
Depreciations are then only on in 2.7. Should be there in base part as well.

Reply:
fixed that.